src/systems/systems/src/hydraulic/linear_actuator.rs

```diff
@@ -646,17 +815,12 @@ impl LinearActuatedRigidBodyOnHingeAxis {
         new_body.actuator_extension_gives_positive_angle =
             Self::initialize_actuator_force_direction(new_body);
```

Comment:
I know this wasn't addressed in this PR, but I don't see why `initialize_actuator_force_direction` would be static. Personally, would prefer having `new_body.initialize_actuator_force_direction()`.

Furthermore, it looks like you are moving `new_body` when you are calling this function. This only works in this case, because you mark it as `#[derive(PartialEq, Clone, Copy)]` meaning you are actually creating a copy.

Reply:
Was initially used while creating object itself thus object was not existing at call time. Now can definitly make this a method for consistency indeed!